code/src/pages/CastDetails.js

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useState } from "react"
+import { useParams, Link } from "react-router-dom"
+import "pages/castdetails.css"
+
+
+export const CastDetails = () => {
+  const { castId } = useParams()
+
+  const [person, setPerson] = useState([])
+  const [actorName, setActorName] = useState("")
+
+  const api_key = "363444609247127238629594b245e069"
+
+  //https://api.themoviedb.org/3/person/192/52fe4210c3a36847f800135f?api_key=363444609247127238629594b245e069&language=en-US
+  useEffect(() => {
+    fetch(`https://api.themoviedb.org/3/person/${castId}/movie_credits?api_key=${api_key}&language=en-US`)
+      .then((res) => res.json())
+      .then((json) => {
+        setPerson(json.cast)
+        console.log(json.cast)
+      })
+  }, [castId])
+
+
+  //https://api.themoviedb.org/3/credit/52fe4210c3a36847f800135f?api_key=363444609247127238629594b245e069
+  useEffect(() => {
+    fetch(`https://api.themoviedb.org/3/credit/52fe4210c3a36847f800135f?api_key=${api_key}`)
+      .then((res) => res.json())
+      .then((json) => {
+        setActorName(json.person)
+        console.log(json.person)
+      })
+  }, [])
+
+  return (
+    <div className="cast-details-top">
+      <h1>Movies with {actorName.name}</h1>
```

Comment:
This doesn't seems to be changed. But your url is correct and from there you get an id. So when you could use the id here to get the name

Reply:
Hej! 
How do you mean? I need to get the credit_Id into the second fetch url but I dont know how to do that..